coriolis/osmorphing/oracle.py

```diff
@@ -34,10 +34,15 @@ def _enable_oracle_repos(self):
                 "http://public-yum.oracle.com/public-yum-ol%s.repo" %
                 major_version)
 
-            self._enable_repos = ["ol%s_software_collections" % major_version,
-                                  "ol%s_addons" % major_version]
+            repos_to_enable = ["ol%s_software_collections" % major_version,
+                               "ol%s_addons" % major_version, ]
+            repos = self._find_yum_repos(repos_to_enable)
         else:
             self._yum_install(['oraclelinux-release-el%s' % major_version])
+            repos_to_enable = ["ol%s_appstream" % major_version,
+                               "ol%s_UEKR6" % major_version]
+            found_repos = self._find_yum_repos(repos_to_enable)
             self._exec_cmd_chroot(
-                "yum config-manager --enable ol%(release)s_appstream "
-                "ol%(release)s_UEKR6" % {"release": major_version})
+                "yum config-manager --enable %s" % " ".join(found_repos))
```

Comment:
Looking through some of the providers it seems that the repos returned by this method are also passed to `_yum_install` (and thus temporarily enabled during the actual package installation, which is preferred to enabling them outright here IMO to prevent modifying the VMs as much as possible during OSMorphing)

Is there any reason/counterexample to still keep the actual enablement command here?

If not, I'd like to request you have this method only return the repo names (and rename the method to something like `_get_oracle_repos`.